load-balancing/elb/common_functions.sh

```diff
@@ -155,11 +155,17 @@ autoscaling_exit_standby() {
         return 1
     fi
 
-    if [ "$instance_state" == "InService" ]; then
+    if [ "$instance_state" == "InService" || "$instance_state" == "Pending:Wait" ]; then
```

Comment:
Can you revert this line change? I'd prefer to have the new block you added catch this state to provide better messaging when this occurs.